lms/djangoapps/course_api/api.py

```diff
@@ -268,3 +269,65 @@ def get_course_run_url(request, course_id):
     """
     course_run_url = reverse('openedx.course_experience.course_home', args=[course_id])
     return request.build_absolute_uri(course_run_url)
+
+
+def get_course_members(course_key, include_students=True, access_roles=None, prefetch_accessroles=False, prefetch_enrollments=False):
```

Comment:
Improve the function name to match it's functionality: `get_course_member_queryset`.

```suggestion
def get_course_member_queryset(course_key, include_students=True, access_roles=None, prefetch_accessroles=False, prefetch_enrollments=False):
```